### DISCUSSION TITLE
[BUG] Error when using Invoke-PnPSiteTemplate where field names or calculation formulas have ampersand (&)

### DISCUSSION THREAD

**Discussion by ITProCorner** [pnp/powershell#1260] [Q&A]
running Invoke-PnPSiteTemplate where field names or calculation formulas have ampersand (&) inside the template returns an error 

Invoke-PnPSiteTemplate: ']' is an unexpected token. The expected token is ';'. Line 1, position 598.

            <Field Type="Calculated" DisplayName="Displ Full Address" EnforceUniqueValues="FALSE" Required="FALSE" Hidden="FALSE" ReadOnly="TRUE" CanToggleHidden="FALSE" ID="{f9541fba-c773-4649-abef-22e5c4dda40b}" Name="Displ_x0020_Full_x0020_Address" StaticName="Displ_x0020_Full_x0020_Address" Format="DateOnly" LCID="1033" ResultType="Text" Percentage="FALSE" SourceID="{{listid:Case Relo List}}" ColName="sql_variant4" RowOrdinal="0">
              **<Formula>=[{fieldtitle:Displacement_x0020_Address}]&amp;", "&amp;[{fieldtitle:Displacement_x0020_City}]&amp;", "&amp;[{fieldtitle:Displacement_x0020_State1}]&amp;" "&amp;[{fieldtitle:Displacement_x0020_Zip}]</Formula>**
            </Field>

or 
 <Field Type="Currency" DisplayName="Agreement Amt F&amp;E" Description="Amount paid 

How do i go about this error? i tried escaping & with no success

**Comment by veronicageek** [comment]
@ITProCorner - Can you update the module to the latest version and report back if the issue persists. Thanks.
